Samples/OAuth2Manager/cpp-winui/TestOAuthInCPlusPlus/StartProgram.cpp

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "App.xaml.h"
+#include "MainWindow.xaml.h"
+#include <winrt/Microsoft.UI.Xaml.h>
+#include <winrt/Windows.Foundation.h>
+#include <winrt/Microsoft.UI.Dispatching.h>
+#include <winrt/Windows.ApplicationModel.h>
+#include <winrt/Microsoft.Windows.AppLifecycle.h>
+#include <winrt/Windows.ApplicationModel.Activation.h>
+
+using namespace winrt;
+using namespace Windows::Foundation;
+using namespace Microsoft::UI::Dispatching;
+using namespace Microsoft::Windows::AppLifecycle;
+using namespace Windows::ApplicationModel::Activation;
+using namespace winrt::TestOAuthInCPlusPlus::implementation;
+
+static Microsoft::UI::Dispatching::DispatcherQueue dispatcherQueue{ nullptr };
+
+void OnActivated(Windows::Foundation::IInspectable const&, AppActivationArguments const& args)
+{
+    if (args.Kind() == ExtendedActivationKind::Protocol)
```

Comment:
May be to good to add a check here for the Uri to confirm that Protocol activation is happening from OAuth, may need a specific Uri for OAuth as well

Reply:
Done.